nx-dev/nx-dev/public/documentation/latest/shared/angular-plugin.md

```diff
@@ -1,5 +1,7 @@
 # Angular Plugin
```

Comment:
I guess this is just a copy and paste of the other file? lol

Reply:
Yeah, this is the one that will actually be displayed on nx.dev as `Latest`. I'm modifying it now so we don't have to way for a new release to have these updates published.